mta-sts.md

```diff
@@ -498,4 +498,34 @@ https://mta-sts.example.com/.well-known/mta-sts.json:
 
 ~~~~~~~~~
 
+# Appendix 2: Sending MTA State Machine
+
+It is helpful to think of the different states of a sending MTA and a receiving
```

Comment:
This works. It might be useful to have a decision tree diagram, but, not sure if diagrams are something commonly used.